### DISCUSSION TITLE
Welcome to `urban_road_filter` Discussions!

### DISCUSSION THREAD

**Discussion by horverno** [jkk-research/urban_road_filter#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KshitizKumarGupta** [comment]
Hey, I am trying to use the algo on single velodyne-32 lidar. But during the parameter-tuning it is crashing and also its giving weird output for curbs. Can u suggest something??

**Comment by horverno** [comment]
hey, @KshitizKumarGupta maybe you can try to adjust the `interval` param which adjusts the LIDAR's vertical resolution. Try it in a launch file, but as far as i remember rqt_reconfigure may also works.
